pkg/types/config.go

```diff
@@ -136,6 +137,7 @@ func (sys *Config) Init() {
 		Name: konfigadm,
 		Vars: make(map[string]interface{}),
 	}
+	sys.Fs = afero.NewOsFs()
```

Comment:
What is this used for ?

Reply:
This is used to interact with the OS filesystem - reading and writing files. In the test code, a memory based file system is used - more like a mock, in the main code, the OS file system (actual fs) is used

Reply:
removed this! 👍